### DISCUSSION TITLE
In App Purchase support for Android

### DISCUSSION THREAD

**Discussion by vanniktech** [fastlane/fastlane#17226] [Ideas]
It would be nice if one could manage in app products using fastlane - similiar to how it's possible on iOS - https://github.com/fastlane/fastlane/issues/1631

**Comment by max-ott** [comment]
It's possible via the Google Play Publishing API (with some limitations): https://developers.google.com/android-publisher/api-ref/rest/v3/inappproducts Not sure if `fastlane` / `supply` supports this already.

**Comment by vanniktech** [comment]
Yeah I know that the API exists; fastlane supply does not support it though.

**Reply by max-ott** [reply]
Feel free to open a PR. I believe the Android user base of `fastlane` is too small that @joshdholtz would start with the implementation of this soon.

**Comment by qalqi** [comment]
Is it possible to support this?

**Reply by vanniktech** [reply]
I wrote something for this: https://github.com/BeLabs/appstatistics/tree/master/inapp-products although there's no documentation but if you run it from the CLI it should auto fill itself as long as you have some products already
